tools/vsimporter/xib2nib/UILabel.cpp

```diff
@@ -94,20 +100,49 @@ void UILabel::InitFromStory(XIBObject* obj) {
         _numberOfLines = atoi(getAttrAndHandle("numberOfLines"));
     }
 
-    if (getAttrib("textAlignment")) {
-        const char* pAlign = getAttrib("textAlignment");
+    const char* lineBreakModeAttributeString = "lineBreakMode";
+    const char* lineBreakModeAttributeValue = getAttrib(lineBreakModeAttributeString);
+    if (lineBreakModeAttributeValue) {
+        if (strcmp(lineBreakModeAttributeValue, "wordWrap") == 0) {
+            _lineBreakMode = 0;
+            getAttrAndHandle(lineBreakModeAttributeString);
+        }
+        else if (strcmp(lineBreakModeAttributeValue, "characterWrap") == 0) {
+            _lineBreakMode = 1;
+            getAttrAndHandle(lineBreakModeAttributeString);
+        }
+        else if (strcmp(lineBreakModeAttributeValue, "clip") == 0) {
+            _lineBreakMode = 2;
+            getAttrAndHandle(lineBreakModeAttributeString);
+        }
+        else if (strcmp(lineBreakModeAttributeValue, "headTruncation") == 0) {
+            _lineBreakMode = 3;
+            getAttrAndHandle(lineBreakModeAttributeString);
+        }
+        else if (strcmp(lineBreakModeAttributeValue, "tailTruncation") == 0) {
+            _lineBreakMode = 4;
+            getAttrAndHandle(lineBreakModeAttributeString);
+        }
+        else if (strcmp(lineBreakModeAttributeValue, "middleTruncation") == 0) {
+            _lineBreakMode = 5;
+            getAttrAndHandle(lineBreakModeAttributeString);
+        }
```

Comment:
>} [](start = 8, length = 1)

Should we print out a warning message that we extracted an unexpected linebreakMode setting?